apps/src/code-studio/components/stageExtras/BonusLevels.jsx

```diff
@@ -15,6 +17,10 @@ const styles = {
     clear: 'both',
     overflow: 'hidden',
   },
+  solutionImage: {
+    border: '1px solid black',
```

Comment:
Nit: can we change this from solid black to something less harsh? #c6cacd is a gray we use in many places.